swagger/swagger-generator/generator-springmvc/src/main/java/org/apache/servicecomb/swagger/generator/springmvc/processor/annotation/RequestHeaderAnnotationProcessor.java

```diff
@@ -32,7 +32,11 @@ public HeaderParameter createParameter() {
 
   @Override
   public String getAnnotationParameterName(Object annotation) {
-    return ((RequestHeader) annotation).name();
+    String name = ((RequestHeader) annotation).name();
+    if (name == null || name.isEmpty()) {
+      name = ((RequestHeader) annotation).value();
```

Comment:
Should consider about RequestParam, CookieValue, etc. And please add integration tests in integration-tests.

Reply:
name will never be null

i'm not sure if it is good to use name() directly 
in very early version, we add similar logic and then delete it
because in that time, some customer use old spring version, that not support name() property.

Reply:
if we decide to add all these new properties, must look at all annotation processors

Reply:
done, springmvc's other annotation has been modified, and read from value first, if it's value is "", read from name